applications/RANSApplication/python_scripts/apply_wall_function_process.py

```diff
@@ -55,10 +56,21 @@ def __init__(self, Model, settings):
                 settings["model_part_name"].GetString())
 
             for node in self.model_part.Nodes:
-                node.Set(KratosMultiphysics.SLIP, True)
                 node.Set(KratosMultiphysics.STRUCTURE, True)
 
             for condition in self.model_part.Conditions:
-                condition.Set(KratosMultiphysics.SLIP, True)
                 condition.Set(KratosMultiphysics.STRUCTURE, True)
```

Comment:
how big is `self.model_part` usually? If it can be big, these loops should be moved to C++.